docs/dependency_review.md

```
@@ -0,0 +1,69 @@
+  dependency_review:
+    uses: masutaka/actions/.github/workflows/dependency_review.yml@main
+    permissions:
+      contents: read
+    with:
+      comment-summary-in-pr: never
+```
+
+## 関連
+
+- [Configuring dependency review - GitHub Docs](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/configuring-dependency-review)
+- https://github.com/actions/dependency-review-action
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)